style.css

```diff
@@ -750,91 +707,111 @@ hr {
   position: relative;
   height: 1;
   width: 100%;
+  margin: 0;
 }
 
-.contact-section {
+.contactblock {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-items: center;
   width: 100%;
-  margin-top: 20%;
   height: auto;
+  margin-top: 8rem;
 }
 
 .contactMe {
   display: flex;
   flex-direction: column;
   align-items: center;
-  width: 100%;
-  height: auto;
+  justify-items: center;
+  max-width: 100%;
+  height: 49rem;
   border-radius: 75px 0 0 0;
   text-align: center;
   background: #6070ff;
   background-image: url(./asset/Contact/contactimg.svg);
   background-repeat: no-repeat;
-  background-position: right;
-  padding-top: 1rem;
+  background-size: 90% 90%;
+  background-position-y: 2rem;
+  background-position-x: 5.5rem;
 }
 
 .contactTxt {
-  margin-top: 19%;
-  max-width: 100%;
+  align-items: center;
+  width: 100%;
+  height: auto;
+  margin-top: 0;
   font-family: 'Poppins', sans-serif;
   font-size: 40px;
   font-weight: 700;
+  line-height: 52px;
   font-style: normal;
   color: #fff;
+  order: 0;
 }
 
 .Contact_body_text {
-  margin: 0;
-  align-items: center;
-  justify-items: center;
-  max-width: 100%;
+  margin-top: 0;
+  width: 100%;
   height: auto;
   font-family: 'Poppins', sans-serif;
   font-size: 20px;
   font-weight: 400;
   font-style: normal;
   line-height: 28px;
+  margin-left: 2rem;
   color: #ebebff;
-  padding: 0 1.5rem;
-  margin-bottom: 2.5rem;
+  order: 1;
+}
+
+.body-text {
+  justify-content: center;
+  align-content: center;
+  width: 90%;
+  height: auto;
+  margin: 0;
 }
 
 .form {
   display: flex;
   flex-direction: column;
-  margin-left: 24px;
-  margin-right: 24px;
+  width: 100%;
+  align-items: center;
+  margin-top: 1rem;
+  order: 2;
 }
 
 #name {
   border-radius: 8px;
   border: 1px #cfd8dc solid;
   background-color: #fff;
   margin-bottom: 20px;
-  padding: 15px 35px;
+  padding: 15px 25px;
+  width: 70%;
+  order: 0;
 }
 
 #mail {
   border-radius: 8px;
   border: 1px #cfd8dc solid;
   background-color: #fff;
-  margin-bottom: 20px;
-  padding: 15px 35px;
+  padding: 15px 25px;
+  width: 70%;
+  margin-bottom: 15px;
 }
```

Comment:
As requested by the previous code reviewer please consider making the border-radius property in accordance with the template.

## Solution:-

Consider replacing the above-used border-radius: 8px with following:-

```
border-radius: 2px;
```